pkg/app/piped/platformprovider/lambda/function.go

```diff
@@ -100,6 +103,11 @@ func (fmp FunctionManifestSpec) validate() error {
 			return fmt.Errorf("architecture is invalid: %w", err)
 		}
 	}
+	if fmp.EphemeralStorage != nil && fmp.EphemeralStorage.Size != 0 {
```

Comment:
This condition is wrong 🤔 If the size value is set to `0` the validate function will be bypass.
```suggestion
	if fmp.EphemeralStorage != nil {
```

Reply:
Thank you!
I fixed these conditions! [fix condition](https://github.com/pipe-cd/pipecd/pull/4680/commits/8af45bb9ffde7b091a96363a0c8c30d2f2f99c61)